infrastructure/github-env.yml

```diff
@@ -103,6 +103,8 @@ Resources:
               - ec2:RevokeSecurityGroupIngress
               - ec2:RevokeSecurityGroupEgress
               - ec2:DescribeSecurityGroupRules
+              - ec2:CreateTags
```

Comment:
Are these extra perms required because the test deploys login nodes?

Reply:
yes